pandas/tests/indexes/test_base.py

```diff
@@ -2289,6 +2289,16 @@ def test_dt_conversion_preserves_name(self, dt_conv):
         index = pd.Index(["01:02:03", "01:02:04"], name="label")
         assert index.name == dt_conv(index).name
 
+    def test_data_frame_value_counts(self):
+        df = pd.DataFrame({'num_legs': [2, 4, 4], 'num_wings': [2, 0, 0]},
+                          index=['falcon', 'dog', 'cat'])
+        assert df.value_counts().equals(pd.Series(data=[2, 1],
```

Comment:
You should be able to use `tm.assert_series_equal(result, expected)` here (should see idiom in other tests

Also this probably belongs in pandas/tests/frame/test_analytics.py

Reply:
Done.